sdk/KoreBuild/modules/frameworkbuild/4-package.targets

```diff
@@ -0,0 +1,64 @@
+<Project>
+  <PropertyGroup>
+    <PackageDependsOn>$(PackageDependsOn);PackageRuntimeStore;PackageMetaPackage</PackageDependsOn>
+  </PropertyGroup>
+
+  <Target Name="PackageRuntimeStore" DependsOnTargets="CompileRuntimeStore">
+    <ZipArchive
+      File="$(ArtifactsDir)runtimestore\$(RuntimeStoreFileName).zip"
+      WorkingDirectory="$(IntermediateDir).rs\"
+      SourceFiles="@(RuntimeStoreFiles)"
+      Condition="'$(IsLinux)' != 'true'" />
+
+    <!-- TODO implement in buildtools. -->
+    <TarballArchive
+      File="$(ArtifactsDir)runtimestore\$(RuntimeStoreFileName).tar.gz"
+      WorkingDirectory="$(IntermediateDir).rs\"
+      Gzip="true"
+      SourceFiles="@(RuntimeStoreFiles)"
+      Condition="'$(IsLinux)' == 'true' OR '$(IsMacOs)' == 'true'" />
+
+    <!-- TODO add other properties to this task for the package manifest. -->
+    <CreateDebianPackage
```

Comment:
Not sure we can build the debian packages right after compilation. We need the dlls in the runtime store to be signed before packaging up the installers.

Reply:
Would this tweak work?
```xml
<PropertyGroup>
   <PackageRuntimeStoreDependsOn>$(PackageRuntimeStoreDependsOn);CompileRuntimeStore;</PackageRuntimeStoreDependsOn>
</PropertyGroup>

<Target Name="PackageRuntimeStore" DependsOnTargets="$(PackageRuntimeStoreDependsOn)">
<!- ... -->
```

Coherence-signed could then do this:
```xml
<PropertyGroup>
   <PackageRuntimeStoreDependsOn>$(PackageRuntimeStoreDependsOn);SignRuntimeStoreBinaries;</PackageRuntimeStoreDependsOn>
</PropertyGroup>

<Target Name="SignRuntimeStoreBinaries" DependsOnTargets="CompileRuntimeStore">
<!- ... -->
```

Reply:
Worth a shot. I'm a little confused with the build steps though, so if you didn't have Coherence-Signed's targets, you'll end up packaging installers with unsigned dlls? How do we guarantee that the coherence-signed targets are always invoked?

Reply:
Yeah, I was just talking about this with Murat. I need to think this piece through more. We should whiteboard it.